compiler/src/compiler.rs

```diff
@@ -110,6 +110,7 @@ impl<F: Field + PrimeField, G: GroupType<F>> Compiler<F, G> {
     }
 
     /// Parses the Leo program file, constructs a syntax tree, and generates a program.
+    #[allow(deprecated)]
```

Comment:
This is not deprecated

Reply:
Yes, I updated it for inner function call